UnityProject/Assets/Scripts/NPC/AI/MobExplore.cs

```diff
@@ -204,6 +368,23 @@ protected virtual void PerformTargetAction(Vector3Int checkPos)
 
 					break;
 				case Target.injuredPeople:
+					var players = MatrixManager.GetAt<PlayerScript>(checkPos, isServer: true);
+					if (players.Count() != 0)
+					{
+						foreach (PlayerScript player in players)
+						{
+							if (player != null && player != recentPatient)
+							{
+								if (!IsEmagged && player.playerHealth.OverallHealth < 75)
+								{
+									player.playerHealth.CirculatorySystem.BloodPool.Add(new ReagentMix(HEALING_REAGENT, 5, 283.15f));
+									Chat.AddCommMsgByMachineToChat(gameObject, "Feel better soon.", ChatChannel.Local, Loudness.NORMAL, broadcasterName: "Medibot: ");
+								}
+								else if (IsEmagged) player.playerHealth.CirculatorySystem.BloodPool.Add(new ReagentMix(HARMFUL_REAGENT, 5, 283.15f));
+								else continue;
+							}
+						}
+					}
```

Comment:
I see you're just following the existing pattern, but this logic and the rest of the code specific to one mob should really be delegated to some other component and not reside in the controller. But the refactor can be for another time : )